requirements/requirements-codestyle.txt

```diff
@@ -1,7 +1,7 @@
 # PEP8 code linting, which we run on all commits.
-flake8==3.8.3
+flake8==3.8.4
 flake8-tidy-imports==4.1.0
 pycodestyle==2.6.0
```

Comment:
Each minor flake8 release only works with one `pycodestyle` release. Imo you can remove this pin, since `flake8` depends on `pycodestyle` - and many other requirements

```suggestion
```